# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1379150365 -7200
#      Sat Sep 14 11:19:25 2013 +0200
# Node ID 9289bb0ff4dd806453993865197ade533793f7d9
# Parent  6690dba6078ae07254d16d6f743e2a1b66eadcb8
io.tst: fix for-loop upper bound after change 3856298f1ff8

diff --git a/test/io.tst b/test/io.tst
--- a/test/io.tst
+++ b/test/io.tst
@@ -318,17 +318,17 @@
 %!error <Invalid call to sprintf> sprintf ()
 %!error <format TEMPLATE must be a string> sprintf (1)
 
 %!test
 %! arch_list = {"native"; "ieee-le"; "ieee-be"};
 %! warning ("off", "Octave:fopen-mode");
 %! status = 1;
 %!
-%! for i = 1:6
+%! for i = 1:3
 %!   arch = arch_list{i};
 %!   for j = 1:4
 %!     if (j == 1)
 %!       mode_list = {"w"; "r"; "a"};
 %!     elseif (j == 2)
 %!       mode_list = {"w+"; "r+"; "a+"};
 %!     elseif (j == 3)
 %!       mode_list = {"W"; "R"; "A"};
